fix(core): fix busy indicator accessibility bugs

aberikashvili opened the pull request:
#### Please provide a link to the associated issue.
Closes #6095

#### Please provide a brief summary of this pull request.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [N/A] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [N/A] update `README.md`
- [N/A] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/SAP/fundamental-ngx?pullRequest=6472) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/SAP/fundamental-ngx?pullRequest=6472) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/SAP/fundamental-ngx?pullRequest=6472) it.</sub>

platon-rov commented on libs/core/src/lib/busy-indicator/busy-indicator.component.ts:
```diff
@@ -58,6 +61,15 @@ export class BusyIndicatorComponent {
     @Input()
     ariaLive: 'assertive' | 'polite' | null = 'polite';
 
+    @Input()
```
Add TypeDoc comments please.

InnaAtanasova replied:
+1

aberikashvili replied:
done

platon-rov reviewed (changes requested):
About this one https://github.com/SAP/fundamental-ngx/issues/6095#issuecomment-906184020

Seems like it's not working properly, I cannot jump to the inner content of the Busy Indicator (table in my case).

<img width="960" alt="Screenshot 2021-09-08 at 15 08 32" src="https://user-images.githubusercontent.com/20265336/132499251-7027c3b5-2838-4498-b9c3-fe9d9dbf83ab.png">

InnaAtanasova commented on libs/core/src/lib/busy-indicator/busy-indicator.component.ts:
```diff
@@ -24,6 +24,9 @@ export type BusyIndicatorSize = 's' | 'm' | 'l';
         '[attr.aria-busy]': 'loading',
         '[attr.aria-live]': 'ariaLive',
         '[attr.aria-label]': 'ariaLabel',
+        '[attr.aria-valuemin]': 'ariaValueMin',
+        '[attr.aria-valuemax]': 'ariaValueMax',
+        '[attr.aria-valuenow]': 'ariaValueNow',
```
Can you explain me how are these related to the a11y and the component? As per this documentation: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-valuemax_attribute

these values are for a range widget such as a slider, spinbutton or progressbar.

aberikashvili replied:
@InnaAtanasova per the issue requirements, please see the issue description https://github.com/SAP/fundamental-ngx/issues/6095
![image](https://user-images.githubusercontent.com/3863691/132646006-6d9cc3d4-f149-49c9-b154-dbe913942606.png)

InnaAtanasova replied:
This is a busy indicator, not a progress bar.

aberikashvili commented:
> About this one [#6095 (comment)](https://github.com/SAP/fundamental-ngx/issues/6095#issuecomment-906184020)
> 
> Seems like it's not working properly, I cannot jump to the inner content of the Busy Indicator (table in my case).
> 
> <img alt="Screenshot 2021-09-08 at 15 08 32" width="960" src="https://user-images.githubusercontent.com/20265336/132499251-7027c3b5-2838-4498-b9c3-fe9d9dbf83ab.png">

@platon-rov I addressed your comments, please check again

droshev reviewed (changes requested):
I think there was a confusion. This is not a progress indicator. This is a busy indicator where the progress is indeterminate. Actually we won't need to do anything here.

aberikashvili commented:
Per @droshev 's [note](https://github.com/SAP/fundamental-ngx/pull/6472#pullrequestreview-751306259), I removed aria attributes from busy-indicator

platon-rov commented:
@aberikashvili great, now I see the busy indicator as the wrapper doesn't interrupt the screen-reader. But for some reason focus is present for that wrapper, could you look into it? Example: https://deploy-preview-6472--fundamental-ngx.netlify.app/#/platform/table

![image](https://user-images.githubusercontent.com/20265336/133984825-984b1634-f8bd-459f-8842-01bd5fa95f98.png)

And there is no focus present without your changes: https://fundamental-ngx.netlify.app/#/platform/table

aberikashvili commented:
@platon-rov I've addressed your latest comment, please check again

platon-rov commented:
Tab stop still present, here are the stops how it was and how it should be. Basically, the busy container shouldn't add the tab stop.

<img width="929" alt="Screenshot 2021-09-21 at 13 13 41" src="https://user-images.githubusercontent.com/20265336/134153889-834c3efd-6c34-4059-b90d-133c8664871b.png">

platon-rov reviewed (approved):
great work!

platon-rov commented:
Please fix travis

platon-rov commented:
@aberikashvili what's the status of this pr? Should qa review to be started?

aberikashvili commented:
> @aberikashvili what's the status of this pr? Should qa review to be started?

I think yes, PR is ready to merge
